addon/components/high-charts.js

```diff
@@ -18,16 +18,17 @@ export default Ember.Component.extend({
   chartOptions: undefined,
   chart: null,
   theme: undefined,
+  callback: undefined,
 
   buildOptions: computed('chartOptions', 'content.@each.isLoaded', function() {
-    let chartOptions = Ember.$.extend(true, {}, get(this, 'theme'), get(this, 'chartOptions'));
-    let chartContent = get(this, 'content.length') ? get(this, 'content') : [{
-      id    : 'noData',
-      data  : 0,
-      color : '#aaaaaa'
-    }];
-
-    let defaults = { series: chartContent };
+    let defaultChartData = [{
```

Comment:
I would revert `L24` to `L31` - https://github.com/airbnb/javascript#variables

Makes things harder to maintain.